app/src/components/Text.jsx

```diff
@@ -0,0 +1,13 @@
+import "./Text.css";
+
+const Text = ({ size = "m", color = "red" }) => {
```

Comment:
this should be an atomic component. you cannot add 3 headers here

this component should show only 1 text per time. then you use it somewhere multiple times

Reply:
also add Header component